specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -5909,7 +5909,7 @@
       "properties": {
         "name": {
           "type": "string",
-          "description": "The sku name."
+          "description": "The sku name. Use **Aligned** for managed disks, and **Classic** for unmanaged disks."
```

Comment:
Refer to the description of storageAccountType under ManagedDiskParameters. In order to be consistent with  documentation I believe this should be updated to 
The sku name. Possible values are: Aligned for managed disks, and Classic for unmanaged disks. Default value is Classic if not specified.

Reply:
I agree with Sameer's comment on adding a default text to let customers know default values.